### PR TITLE
[simd/v3i]: Implement v128 bitwise instructions

### PR DESCRIPTION
**Pull request by haoyu-zc**
Tested by `make v3i -j && bin/spectest.v3i -tk test/regress/simd/v128.xor.bin.wast | progress`

### CODE REVIEW

File: src/engine/Value.v3 [hunk 1]
```diff
@@ -135,6 +135,7 @@ component Values {
 	def v_d(v: Value) -> u64 { return Value.F64.!(v).bits; }
 	def v_l(v: Value) -> i64 { return i64.view(Value.I64.!(v).val); }
 	def v_w(v: Value) -> u64 { return Value.I64.!(v).val; }
+	def v_v128(v: Value) -> (u64, u64) { return (Value.V128.!(v).low, Value.V128.!(v).high); }
```
Comment (titzer) [inline src/engine/Value.v3:138]: Can you factor out the cast here to a local variable?

File: src/engine/v3/V3Interpreter.v3 [hunk 2]
```diff
@@ -910,6 +910,7 @@ component V3Interpreter {
 				var high = codeptr.read_u64();
 				push(Value.V128(low, high));
 			}
+			V128_XOR => do_vv_v(V3Eval.V128_XOR);
```
Comment (titzer) [inline src/engine/v3/V3Interpreter.v3:913]: Please go ahead and do all the `i64x2` pure operations to save on the number of PRs.

File: src/engine/Value.v3 (hunk 1 above)
Comment (haoyu-zc) [reply src/engine/Value.v3:138]: Yes sure.

File: src/engine/v3/V3Interpreter.v3 (hunk 2 above)
Comment (haoyu-zc) [reply src/engine/v3/V3Interpreter.v3:913]: Got it. Just want to make sure I'm on the right track. Further PRs will have multiple realizations as before.

**Comment by haoyu-zc** [comment]
Tested by: `make v3i -j && bin/spectest.v3i -tk wasm-spec/src/spec/simd/simd_bitwise.bin.wast`

**Review by titzer — APPROVED** [review]
lgtm

**Pull request merged**